main.py

```diff
@@ -2,6 +2,29 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [("George", "Klooney"),
```

Comment:
*I've told you about that in previous comment

Reply:
> Why didn't you move assigning variables to the start of the function?
> by that you'll improve code readability and ensure that variables are defined and initialized in a logical order, making it easier to understand and maintain the code
> But if you don't think so, you can share your opinion

when I learned C++, in that case was opposite opinion about assigning variables, so that's just my habit and I forgot to replace it, sorry